Kms/README.md

```diff
@@ -0,0 +1,102 @@
+# Cloud KMS for PHP
+
+> Idiomatic PHP client for [Cloud KMS](https://cloud.google.com/kms).
+
+[![Latest Stable Version](https://poser.pugx.org/google/cloud-kms/v/stable)](https://packagist.org/packages/google/cloud-kms) [![Packagist](https://img.shields.io/packagist/dm/google/cloud-kms.svg)](https://packagist.org/packages/google/cloud-kms)
+
+* [API documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/cloud-kms/latest/kms/readme)
+
+**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googlecloudplatform/google-cloud-php). Any
+support requests, bug reports, or development contributions should be directed to
+that project.
+
+### Installation
+
+To begin, install the preferred dependency manager for PHP, [Composer](https://getcomposer.org/).
+
+Now to install just this component:
+
+```sh
+$ composer require google/cloud-kms
+```
+
+Or to install the entire suite of components at once:
+
+```sh
+$ composer require google/cloud
+```
+
+This component supports both REST over HTTP/1.1 and gRPC. In order to take advantage of the benefits offered by gRPC (such as streaming methods)
+please see our [gRPC installation guide](https://cloud.google.com/php/grpc).
+
+
```

Comment:
Sorry to nag on this again, extra newline here. I've opened a PR to fix the template: https://github.com/GoogleCloudPlatform/google-cloud-php/pull/1144 :)

Reply:
Done